Sources/ApolloCodegenLib/Glob.swift

```diff
@@ -82,12 +83,12 @@ public struct Glob {
       }
     }
 
-    return Set<String>(includeMatches).subtracting(excludeMatches)
+    return OrderedSet<String>(includeMatches).subtracting(excludeMatches)
```

Comment:
remind me, why are we making this an ordered set? I would think ordered set has worse perf than a vanilla set.

Reply:
It does affect performance but I expect it aids predictability and testability where items are always in the same order. I encountered that recently when dealing with input object fields.

Reply:
Ah, yeah, we've also had requests for consistent ordering of fields so that makes sense

Reply:
Yeah, this has to be ordered because it determines the order that we pass the graphQL documents into the JS compiler, which affects the order of collecting the referenced types. The schema file was rendering with objects in different orders before this.